content/en/docs/instrumentation/js/getting-started/browser.md

```diff
@@ -42,24 +50,37 @@ To create traces in the browser, you will need `@opentelemetry/sdk-trace-web`, a
 
 ```shell
 npm init -y
-npm install --save @opentelemetry/api @opentelemetry/sdk-trace-web @opentelemetry/instrumentation-document-load @opentelemetry/context-zone
+npm install  @opentelemetry/api @opentelemetry/sdk-trace-web @opentelemetry/instrumentation-document-load @opentelemetry/context-zone
 ```
 
 ### Initialization and Configuration
 
-Create a empty file called `document-load.js` and add the following code to your html right before the body end tag:
+If you are coding in TypeScript, then run the following command: 
+```shell
+tsc --init
+```
+Create an empty code file named `document-load` with a `.ts` or `.js` extension, as appropriate, based on the language you've chosen to write your app in. Add the following code to your HTML right before the `</body>` closing tag:
 
-```html
+{{< ot-tabs TypeScript JavaScript >}}
+
+{{< ot-tab lang="html">}}
+<script type="module" src="document-load.ts"></script>
```

Comment:
I don't think browsers are able to load typescript source codes directly. For both JavaScript and TypeScript projects, here the src attribute value should be `document-load.js` (the output of tsc compilation).

Reply:
If I remember correctly parel (the build tool used in the example) will infer the type of the file from that and then do a proper compile and replacement. @kaiwalyakoparkar have you run through that example with TS and can verify, if not let's try this out.

Reply:
Hey @svrnm I did run these files and can confirm that the output I received is according to the docs

Reply:
IMO it's quite confusing as parcel isn't mentioned until after the code samples. If you want to depend on bundler-specific behavior I would at least call it out at the top of the document that the samples will require some particular build process or dependency.

Reply:
@dyladan fair point! That one is on me, I wrote that tutorial initially. Let's have this in a separate (follow-up) issue

Reply:
I filed https://github.com/open-telemetry/opentelemetry.io/issues/1989 for this issue